mockredis/client.py

```diff
@@ -221,7 +222,9 @@ def do_expire(self):
         """
         Expire objects assuming now == time
         """
-        for key, value in self.timeouts.items():
+        # Deep copy to avoid RuntimeError: dictionary changed size during iteration
+        _timeouts = deepcopy(self.timeouts)
+        for key, value in _timeouts.items():
```

Comment:
looks good to me. I was bit confused with PR a bit (missed python 3 ;)). I got it now. Thank you for your PR 👍